### PR TITLE
Issue #257 enable CORS

### PR DESCRIPTION
**Pull request by kishansagathiya**

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @kishansagathiya for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

File: internal/proxy/proxy.go [hunk 1]
```diff
@@ -132,9 +132,16 @@ func NewProxy(tenant *clients.Tenant, wit clients.WIT, idler clients.IdlerServic
...
+func enableCORS(w http.ResponseWriter) {
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Access-Control-Allow-Methods", "POST, GET, OPTIONS, PUT, DELETE")
+	w.Header().Set("Access-Control-Allow-Headers", "Accept, Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization")
+}
+
 //Handle handles requests coming to the proxy and performs action based on
 //the type of request and state of Jenkins.
 func (p *Proxy) Handle(w http.ResponseWriter, r *http.Request) {
+	enableCORS(w)
```
Comment (sbose78) [inline internal/proxy/proxy.go]: will it modify `w` or a copy of `w` ?

**Comment by alien-ike** [comment]
Hey @sbose78! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer or pull request approver, but not pull request creator for this command to take an effect.

**Pull request merged**